.github/workflows/test.yml

```diff
@@ -38,6 +38,6 @@ jobs:
     steps:
       - uses: actions/checkout@v1
       - name: SwiftLint
-        uses: docker://norionomura/swiftlint:0.53.0
+        uses: docker://norionomura/swiftlint:0.55.1
```

Comment:
This makes me want to use [mise](https://mise.jdx.dev) even more. We specify the version here for GitHub Actions, but we don't and can't specify the version for Homebrew. So if a new release of SwiftLint happens tomorrow and someone installs SwiftLint tomorrow with Homebrew, they might have warnings we don't have in CI.

With mise, we can specify the version to install and use:

```
% mise install swiftlint@0.55.1
% mise use swiftlint@0.55.1
```

And it's written to a file that lives in our repo. So everyone can know what version and get that version.

Reply:
hm ok. I'm game to try it out. It seems like everyone has `brew` and making people install another package manager feels like it could be a bigger ask than just dealing with occasional brew issues. But I do like version pinning. We could add the mise files to the repo for starters and see how it works for us before recommending it for others.